ateam_msgs/srv/GetFirmwareParameter.srv

```diff
@@ -0,0 +1,18 @@
+# Request
+int32 robot_id
+int8 parameter_id
+
+int8 PARAM_ID_VEL_PID_X=0
+int8 PARAM_ID_VEL_PID_Y=1
+int8 PARAM_ID_ANGULAR_VEL_PID_Z=2
+int8 PARAM_ID_VEL_CGKF_ENCODER_NOISE=3
+int8 PARAM_ID_VEL_CGKF_GYRO_NOISE=4
+int8 PARAM_ID_VEL_CGKF_PROCESS_NOISE=5
+int8 PARAM_ID_VEL_CGFK_INITIAL_COVARIANCE=6
+int8 PARAM_ID_VEL_CGKF_K_MATRIX=7
```

Comment:
this is missing parameters 8, 9, 10 vel/accel limits. That may have gotten added since your last submodule pull

Reply:
good catch. fixed.